5.5/root/usr/share/container-scripts/mysql/scl_enable

```diff
@@ -1,3 +1,3 @@
 # This will make scl collection binaries work out of box.
 unset BASH_ENV PROMPT_COMMAND ENV
-source scl_source enable mysql55
+source scl_source enable ${ENABLED_COLLECTIONS}
```

Comment:
i understand the variable above now, but don't understand the intention of it... do you want to allow people enabling additional collections when using mysql as a base image for something?